packages/itwinui-css/src/side-navigation/side-navigation.scss

```diff
@@ -124,7 +128,7 @@
     }
 
     &.iui-expand {
-      height: $iui-line-height;
+      height: var(--iui-size-l);
```

Comment:
```suggestion
      height: var(--iui-size-l);
      min-height: unset;
```

We need to remove the min-height (or change it somehow else) because it looks much larger than it was previously.
![Screen Shot 2022-08-12 at 11 59 42 AM](https://user-images.githubusercontent.com/849817/184395856-8b87ba1c-e659-4bab-87c8-afe9f413e563.png)